dbms/src/Flash/Management/tests/gtest_manual_compact.cpp

```diff
@@ -117,7 +117,7 @@ class BasicManualCompactTest
 
     DM::tests::DMTestEnv::PkType pk_type;
 
-    [[maybe_unused]] Poco::Logger * log;
+    Poco::Logger * log;
```

Comment:
Use `MAYBE_UNUSED_MEMBER` or it may get error under mac os. https://github.com/pingcap/tiflash/pull/4700/files
Or just remove this variable since we don't need it?